ddmrp/tests/test_ddmrp.py

```diff
@@ -765,6 +765,22 @@ def test_24_purchase_link(self):
         self.assertIn(pol.buffer_ids, self.buffer_purchase)
         self.assertEqual(self.buffer_purchase.procure_recommended_qty, 0)
 
+    def test_25_auto_procure(self):
+        pol = self.pol_model.search([("product_id", "=", self.product_purchased.id)])
+        self.assertFalse(pol)
+        self.assertGreater(self.buffer_purchase.procure_recommended_qty, 0)
+        self.buffer_purchase.auto_procure = True
+        self.buffer_purchase.auto_procure_option = "stockout"
+        self.buffer_purchase.cron_actions()
+        pol = self.pol_model.search([("product_id", "=", self.product_purchased.id)])
+        self.assertFalse(pol)  # Buffer is not in stockout.
```

Comment:
Would be great to have a test covering this case as well.

Reply:
@grindtildeath Done, I combined it with auto-nfp update for also test both features combined.